test/unit/tab-indicator/index.test.tsx

```diff
@@ -135,7 +135,7 @@ test('child custom element should have content classes', () => {
 test('#componentWillUnmount destroys foundation', () => {
   const wrapper = shallow<TabIndicator>(<TabIndicator />);
   const foundation = wrapper.instance().foundation;
-  foundation.destroy = td.func();
+  foundation.destroy = td.func<() => null>();
```

Comment:
oh wow - i've been using the coereceForTesting function, but this is a lot cleaner and shorter! :)